article/_posts/2021-03-19-DiminishingUncertaintyAL.md

```diff
@@ -0,0 +1,134 @@
+SVGD uses entropy loss). 
+
+## Uncertainty estimation 
+Authors use an entropy based epistemic (model) uncertainty for every 3D voxel defined by 
+
+![](/article/images/DiminishingUncertaintyAL/eq2.jpg)
+
+this is computed for every particle $$q$$ and every sample in the unlabeled set. 
+
+Authors also add a second term to the score of a sample based on mutual information with samples from the labeled set.
+
+![](/article/images/DiminishingUncertaintyAL/eq3.jpg)
+
+where $$P(x^{\mathcal{T}}_i, x^{\mathcal{U}}_j)$$ is the joint probability and $$P(x^{\mathcal{T}}_i)$$, 
+$$P(x^{\mathcal{U}}_j)$$ are marginal probabilities with respect to the histograms of the two images. 
+
+
+
+
+The epistemic uncertainty and mutual information are normalized and the final score is calculated according to: 
+
+![](/article/images/DiminishingUncertaintyAL/eq4.jpg)
+
+The mutual information term is subtracted because a lower score for mutual information indicates more diversity. 
+
+
+## Deleting samples 
+Contrary to other active learning methods, the authors propose leaving samples in the Unlabeled pool once they are 
+labeled. Therefore, if the model is still uncertain about this sample in subsequent steps, it can be sampled once again 
+but will not require more labeling. 
+
+# Data
+
+Authors test their method on 2 datasets:
+* Pancreas and tumor segmentation based on 3D computed tomography (CT) volume
+    * Samples processed by cubic 48x48x48 patches
+    * Down-sampled to 4.0 mm isotropic resolution 
+    * Data clipped in the range -87 to 199 (Hounsfield units) and normalized to [0, 1].
+* Hippocampus segmentation of two different regions of interest based on T1-weighted magnetic resonance imaging (MRI)
+    * Samples processed by 64x64x64 volumes 
+    * Normalized with clip range used was 0 to 2048.
+
+# Results
+The following table shows the hyper-parameters used for the active learning experiments. 
+
+| Hyper-parameters                               | Hippocampus     | Pancreas        |
+|------------------------------------------------|-----------------|-----------------|
+| Learning rate                                  | 0.001           | 0.0004          |
+| Batch size (volumes)                           | 8               | 8               |
+| Active iterations                              |       40        |      40         |
+| Number of models (SGVD)                        |        5        |      5          |
+| Training steps per active step                 | 1500            | 10000           |
+| Set size (subjects) (init train/U/val/test)    | 10/153/50/50    | 20/201/30/30    |
+| Queries per active step                        |       1         |        5        |
+
+
+Authors compare different combinations of their methods with standard ensemble methods. 
+All experiments were performed with 5 different random seeds.
+
+![](/article/images/DiminishingUncertaintyAL/tab1.jpg)
```

Comment:
Not sure if I understand this: does the percentage of training data vary because every method evaluates and pools differently samples that the method considers being most uncertain?

Reply:
Each model is trained for 40 active iterations, the data in the table indicates the amount of data and results for the best step according to validation dice. This is partially explained in the table's caption. Would you like a more explicit description ?

Reply:
Maybe it's my own limited knowledge about comparing active learning methods, but comparing methods using different percentages of the training data is not something you would do unless you are trying to demonstrate that a given method with less available data outperforms another one. But as said, it's maybe the way in each method picks the most interesting data that gives rise to those numbers.

Reply:
OK, understood after a conversation with Thierry. Thanks.